.github/workflows/publish-gh-pages.yml

```
@@ -0,0 +1,14 @@
+# This is a placeholder workflow only. Its purpose is for manual runs to show up
+# in the GitHub web UI. It is not used for any automated runs.
+name: Publish site
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  placeholder:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Placeholder step to have workflow included in the GitHub web UI
+        run: echo "Placeholder step to have workflow included in the GitHub web UI
```

```suggestion
        run: echo "Placeholder step to have workflow included in the GitHub web UI"
```

would it be helpful to print a message indicating that the actual workflow definition is maintained on the `gh-pages` branch? maybe also update the comments to that effect as well.

---

How does that look now? @edgchen1 @faxu